client/src/Features/AssignmentLinks/AssignmentLinkEditor.tsx

```diff
@@ -34,12 +52,15 @@ const AssignmentLinkEditorInner = ({
         value={link?.displayText || ''}
         styles={themedClassNames(textFieldBaseStyle)}
         onChange={(_e, newValue) => onChangeTextField('displayText', newValue || '')}
+        onGetErrorMessage={(value) => LinkTextValidation(value, 500)}

```

Comment:
I'd refrain from using magic numbers in the invocation. Instead, please consider creating a const variable and using that directly inside the function. That should also help refactor this code as: -
```suggestion
        onGetErrorMessage={LinkTextValidation}
```